apps/main-site/src/pages/browse.tsx

```diff
@@ -0,0 +1,21 @@
+import { BrowseCommunitiesPage } from '@answeroverflow/ui/src/components/pages/BrowseCommunitiesPage';
+import type { ServerPublic } from '~api/router/server/types';
+import { trpc } from '@answeroverflow/ui';
+
+export default function BrowseCommunitiesPageWeb({
+	servers,
+}: {
+	servers: ServerPublic[];
+}) {
+	return <BrowseCommunitiesPage servers={servers} />;
+}
+
+export function getServerSideProps() {
+	const servers = trpc.servers.getAllServers.useQuery().data;
```

Comment:
Now that app dir is stable, lets write this in app dir just using a get all servers fetch, this doesnt need to go thru tRPC

Reply:
Done! :+1: